whac-a-mole.py

```diff
@@ -3,6 +3,10 @@
 from time import *
 import threading
 import random
+import winsound

```

Comment:
`winsound` is only supported on Windows, so it's not running on my Linux machine. To make it cross-platform, please consider a cross-platform module.
You can check out `playsound`, `snack sound kit` and many more.

Reply:
okkay, will do both soon